### PR TITLE
Ch 73 stylize the search page

### PR DESCRIPTION
**Pull request by timDeHof**
Stylized the search page to make it similar to the figma design

### CODE REVIEW

**Comment by joekotvas** [comment]
On the deploy preview here, it looks like the front page hero section has lost its yellow wave.

File: src/assets/Recipe Photo.png
Comment (joekotvas) [inline src/assets/Recipe Photo.png:1]: Probably best to use a regular image with sharp corners, then round them with CSS.

File: src/assets/brand/sweet-peppers/png/index.js
Comment (joekotvas) [inline src/assets/brand/sweet-peppers/png/index.js:1]: The sweet-peppers folder has been made obsolete by the code inside bell-peppers.

File: src/assets/brand/sweet-peppers/psd/index.js
Comment (joekotvas) [inline src/assets/brand/sweet-peppers/psd/index.js:1]: The sweet-peppers folder has been made obsolete by the code inside bell-peppers. Also the PSD was removed.

File: src/assets/brand/sweet-peppers/webp/index.js
Comment (joekotvas) [inline src/assets/brand/sweet-peppers/webp/index.js:1]: The sweet-peppers folder has been made obsolete by the code inside bell-peppers.

File: src/assets/index.js
Comment (joekotvas) [inline src/assets/index.js:1]: Best to avoid spaces in file names.

File: src/assets/HomePage/svgWave.jsx
Comment (joekotvas) [inline src/assets/HomePage/svgWave.jsx:1]: I don't think the yellow default value has a function here; there is always a current color inherited from the containing page. Also, as of right now on the front page the currentColor is transparent; we'll need to update that.

File: src/features/recipes/components/feature-of-the-day/feature-of-the-day.jsx
Comment (joekotvas) [inline src/features/recipes/components/feature-of-the-day/feature-of-the-day.jsx:1]: We probably need more top padding on this.

File: src/pages/Search.jsx [hunk 1]
```diff
@@ -10,29 +14,47 @@ const Search = () => {
...
-          <Heading level="h2" variant="watermelon" className="text-center">
-            YumYum Time!!
-          </Heading>
-          <p className="text-xl font-semibold text-center font-kalam text-lava-950">
-            Add Ingredients Here and We Will Do Our Magic!
-          </p>
-          <div className="w-full mx-auto">
-            <SearchBox searchTerm={searchTerm} onSearch={handleSearch} />
+      <SvgComponent className="fill-sky-300" />
+      <div className="flex flex-col flex-shrink-0 w-full lg:relative bg-gradient-Search">
```
Comment (joekotvas) [inline src/pages/Search.jsx]: We need horizontal padding here.

**Review by joekotvas — CHANGES_REQUESTED** [review]
Great progress! Here are a few notes.

**Review by Greimil — APPROVED** [review]

File: src/assets/HomePage/svgWave.jsx
Comment (timDeHof) [reply src/assets/HomePage/svgWave.jsx:1]: fixed: now add className 'fill-{color}' to change wave color

**Review by joekotvas — APPROVED** [review]
Looks great!

**Pull request merged**